### ISSUE TITLE
Setuptools CVE-2022-40897

### ISSUE THREAD

**Issue by matthawley** [locustio/locust#2986]
### Prerequisites

- [X] I am using [the latest version of Locust](https://github.com/locustio/locust/releases/)
- [X] I am reporting a bug, not asking a question

### Description

Version 2.32.2 of locust via the usage of the python 3.11.0-slim base image reports the usage of setuptools@65.5.0 which contains the vulnerability https://github.com/advisories/GHSA-r9hx-vwmv-q579

We are requesting locust to upgrade to a newer version of the python base image to a newer one (3.12.0-slim or 3.13.0-slim) to resolve this issue. Referencing #2761

### Command line

n/a

### Locustfile contents

```python3
n/a
```

### Python version

3.11

### Locust version

2.32.2

### Operating system

Linux

**Comment by cyberw** [comment]
IIRC there was some reason we didnt already update to 3.12, but I've added a specific dependency on setuptools.

Will make a release some time next week. Thanks for reminding me!

**Comment by matthawley** [comment]
@cyberw Unfortunately, it doesn't seem as if this works - docker scout still detects v65.5.1 as it's directly referenced/installed in the [Python 3.11 docker container](https://github.com/docker-library/python/blob/2589112d847139587507c8071a80b07134a4b372/3.11/alpine3.20/Dockerfile#L122C2-L128C5).

```
pkg:pypi/setuptools@65.5.1

    x HIGH CVE-2024-6345 [Improper Control of Generation of Code ('Code Injection')]
      https://scout.docker.com/v/CVE-2024-6345
      Affected range : <70.0.0
      Fixed version  : 70.0.0
      CVSS Score     : 8.8
      CVSS Vector    : CVSS:4.0/AV:N/AC:L/AT:P/PR:N/UI:A/VC:H/VI:H/VA:H/SC:N/SI:N/SA:N
```
